PersonApi/V1/Controllers/BaseController.cs

```diff
@@ -16,10 +16,13 @@ public BaseController()
 
         public string GetCorrelationId()
         {
-            if (HttpContext.Request.Headers[CorrelationConstants.CorrelationId].Count == 0)
```

Comment:
Would you please be able to explain the reason behind making these changes if it was approved in previous PR?

Reply:
See my comment above.